server/views/index.jade

```diff
@@ -16,10 +16,7 @@ block content
             p.help= t('index noscript message')
 
 block js
-    script(src="/scripts/vendor.js")
     if env
         script.
-            require.register('env', function(exports, require, module) {
-              module.exports = !{JSON.stringify(env)}
-            })
-    script(src="/scripts/app.js", onload="require('initialize');")
+            ENV = !{JSON.stringify(env)};
+    script(src="/app.js" defer=true)
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/HTML/Using_the_application_cache advises to avoid appcache, as it is deprecated and can be a pain to use/debug. Service Worker is recommended now.